graphql-client-codegen/build.gradle.kts

```diff
@@ -1,8 +1,42 @@
+import com.strumenta.antlrkotlin.gradle.AntlrKotlinTask
+
 plugins {
     kotlin("jvm")
-    antlr
+    id("com.strumenta.antlr-kotlin") version "1.0.0"
 }
 
 dependencies {
-    antlr("org.antlr:antlr4:4.13.2")
+    implementation("com.strumenta:antlr-kotlin-runtime:1.0.0")
+}
+
+sourceSets {
+    main {
+        kotlin.srcDirs("kotlin", "build/generatedAntlr")
```

Comment:
I don't think the `kotlin` directory is necessary here, is it?

Reply:
since the default ist src/main/kotlin for me this gave an error